app/src/components/widgets/multiLevelWidgets/cumulativeTrendChart/cumulativeTrendChart.jsx

```diff
@@ -144,6 +173,15 @@ export class CumulativeTrendChart extends PureComponent {
     },
   ];
 
+  getDefaultParamsWidget = (filterId) => ({
```

Comment:
`getDefaultParamsWidget` sounds unclear. Please rename it to `getDefaultLinkParams` or `getCommonLinkParams`.